### PR TITLE
Peggy2 docs

### PR DESCRIPTION
**Pull request by Brando753**
Need team sign off on current docs.

### CODE REVIEW

File: docs/peggy/docs/Concepts.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+monitor the Sifchain blockchain for log events regarding transactions; when it sees either an EventTypeLock or EventTypeBurn the Witness constructs a prophecy and signs the prophecy with its EVM private key. The Witness
+then submits the signed prophecy for the sequence number of that transaction into the virtual mempool which is a standard cosmos keeper. The relayer watches the virtual
+mempool. When it observes enough witnesses have submitted prophecies for a transaction to complete, it will then relay the prophecy with the aggregated signatures of those witnesses to the BridgeBank of the EVM chain
+for this transaction. After about a month (based upon block height), the prophecies are cleared from the cosmos keeper.
+
+## Token Denom Hashing for EVM-native assets
+
+To uniquely identify EVM-native assets such as the native currency or ERC20 tokens, we internally use a denom for
+the relayers and sifnode banking modules. The EVM Native denom is a formatted string generated by concatenating `sifBridge`
+followed by the originating internal EVM chain ID as a base 10 number consisting of 4 characters (0000~9999) , followed by
```
Comment (smartyalgo) [inline docs/peggy/docs/Concepts.md:25]: Do you mean network descriptor here? Instead of evm chain id

File: docs/peggy/docs/SmartContracts.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- Then, in order to process a prophecy claim, we check if the resulting public key is a known relayer public key. If it is, we add its validation power to the prophecy.
+
+## BridgeBank
+- lock()
+
+## TokenRegistry
+- @TODO@ Decribe usage, essential fields and methods
+
+## BridgeToken
+- @TODO@ Decribe usage, essential fields and methods
```
Comment (jzvikart) [inline docs/peggy/docs/SmartContracts.md:36]: Is this file still needed since there is "contracts.md"? If not, there are some links still pointing to this file and we should probably change them.

**Comment by smartyalgo** [comment]
Do we need `docs/peggy/docs/.nojekyll` ?

File: docs/peggy/docs/glossary.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Glossary
```
Comment (smartyalgo) [inline docs/peggy/docs/glossary.md:1]: - Missing EthBridgeClaim
- ProphecyClaim is sif->evm only, 

I'll Add those changes

File: docs/peggy/docs/Components.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+Relayers and witnesses are stateless and will, upon startup, query sifnode for the last sequence number that the specific
+relayer/witness has handled from bridgebank. The relayer/witness will then process all events from the block height of
+this sequence number on.
+
+## Witness
+
+
+
+## Relayer
```
[the remark this answers is not in the document]
Comment (Brando753) [reply docs/peggy/docs/Components.md:16]: No its not, deleting it shortly.

File: docs/peggy/docs/SmartContracts.md (hunk 2 above)
Comment (Brando753) [reply docs/peggy/docs/SmartContracts.md:36]: Deleting the directory, will update links.

**Review by smartyalgo — APPROVED** [review]
